test/common/config/delta_subscription_test_harness.h

```diff
@@ -51,12 +52,14 @@ class DeltaSubscriptionTestHarness : public SubscriptionTestHarness {
       xds_context_ = std::make_shared<Config::XdsMux::GrpcMuxDelta>(
           std::unique_ptr<Grpc::MockAsyncClient>(async_client_), dispatcher_, *method_descriptor_,
           random_, stats_store_, rate_limit_settings_, local_info_, false,
-          std::make_unique<NiceMock<MockCustomConfigValidators>>());
+          std::make_unique<NiceMock<MockCustomConfigValidators>>(),
+          /*xds_config_tracker=*/XdsConfigTrackerOptRef());
```

Comment:
Why is this not providing a mock object?

Reply:
Emmm, an `OptRef` of an abstract class is okay for the test, right? We are testing it at least from the integration test to verify the extension method is called.